library/bignum.c

```diff
@@ -359,6 +380,7 @@ int mbedtls_mpi_set_bit( mbedtls_mpi *X, size_t pos, unsigned char val )
 size_t mbedtls_mpi_lsb( const mbedtls_mpi *X )
 {
     size_t i, j, count = 0;
+    MPI_VALIDATE_RET( X != NULL );
```

Comment:
Got a conversion warning here, this returns a size_t rather than an int.

Reply:
Thanks - fixed!